pom.xml

```diff
@@ -71,7 +71,7 @@
 		<dependency>
 			<groupId>commons-fileupload</groupId>
 			<artifactId>commons-fileupload</artifactId>
-			<version>1.3.2</version>
+			<version>1.3.3</version>
```

Comment:
I think this is fine, the file upload is done for sis import flow. That flow is widely used by admins for creating sections, adding users to the sections in the course. This change should be released to production ASAP, I guess